cfpb/cfgov-refresh
# Restore API Program null values

The original EFIP API delivered a string value of "None" for missing data
provided by schools. The proper json payload should have been `null`,
and we fixed this last month, but unfortunately the EFIP tool tests 
for the "None" string, so it was a breaking change.

We want to change the EFIP code as little as possible for now, so our fix
is to restore the original API behavior.

EFIP changes are likely in the near future, and we can address this when
the legacy tool is revisited.

## Testing

- First, go to this [recent live disclosure](https://www.consumerfinance.gov/paying-for-college2/understanding-your-financial-aid-offer/offer/?iped=154022&pid=BAENG&totl=70950&tuit=14160&hous=7109&book=1150&tran=500&othr=4743&pelg=6345&schg=&stag=&othg=&mta=&gib=&wkst=0&ppl=&parl=&perl=&subl=3500&unsl=6000&gpl=&prvl=&prvi=&prvf=&insl=&insi=&inst=&oid=D3DAC50D5CB8671870A32E6518472A74952C9849#info-right), scroll to the bottom and click through to "Step 2." There you should see a graduation rate chart showing 0%. This should be 8%.

- Scroll to the bottom of Step 2 and check the loan default rate chart. I also shows 0%, instead of 14%.

- Check out this branch, and call up the [same disclosure](http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/offer/?iped=154022&pid=BAENG&totl=70950&tuit=14160&hous=7109&book=1150&tran=500&othr=4743&pelg=6345&schg=&stag=&othg=&mta=&gib=&wkst=0&ppl=&parl=&perl=&subl=3500&unsl=6000&gpl=&prvl=&prvi=&prvf=&insl=&insi=&inst=&oid=D3DAC50D5CB8671870A32E6518472A74952C9849#info-right) locally. When you click through,
you should see a grad rate of 8% and a loan default rate of 14%.
